ethereum.star

```diff
@@ -14,6 +14,10 @@ def run(plan, args):
                     "l1_preallocated_mnemonic"
                 ],
             },
-            "additional_services": [],
+            "additional_services": [
+                "blockscout",  # block explorer
+                "dora",  # beaconchain explorer
+                "el_forkmon",  # fork monitor tool
```

Comment:
My preference would be to make these configurable via `params.yml` especially since blockscout could be quite heavy.

Reply:
Addressed in [chore: make l1 additional services configurable via params.yml](https://github.com/0xPolygon/kurtosis-cdk/pull/74/commits/e5e717bd9883eebd6e0044947633f46fa824000d)